### PR TITLE
feat(client): add a link to create message broker in service connections page

### PR DESCRIPTION
**Pull request by Meyazhagan**
Close: #6214 

### Summary
🆕🎨🚀

Added a feature to create a new message broker from the service connections page. Modified the `EmptyState` component to support custom actions.

> _When the service connections are few_
> _The `EmptyState` component will do_
> _But to make it more nice_
> _We added a slice_
> _Of `children` with a button to view_

### Walkthrough
*  Add an optional `children` prop to the `EmptyState` component, which allows rendering additional elements inside the empty state container ([link](https://github.com/amplication/amplication/pull/6874/files?diff=unified&w=0#diff-3acd4b227dadf1a6d2acdeb18478c46d1b75247847b4af5a817ded2dde7e59b9L7-R17))
* Import the `useHistory` hook from `react-router-dom` in the `ServiceConnectionsPage` component, which allows navigating to different routes programmatically ([link](https://github.com/amplication/amplication/pull/6874/files?diff=unified&w=0#diff-02c2e232d5e66303434e92c2e77cd64605c3929643b6f553457884d3a0f9d719L2-R2))
* Import the `Button` component and its related enums from `@amplication/ui/design-system` in the `ServiceConnectionsPage` component, which allows rendering a styled button with an icon ([link](https://github.com/amplication/amplication/pull/6874/files?diff=unified&w=0#diff-02c2e232d5e66303434e92c2e77cd64605c3929643b6f553457884d3a0f9d719R13-R17))
* Add the `history`, `currentWorkspace`, and `currentProject` variables to the `ServiceConnectionsPage` component, using the `useHistory` hook and the `AppContext`, which store the current browser history object, the current workspace object, and the current project object, respectively ([link](https://github.com/amplication/amplication/pull/6874/files?diff=unified&w=0#diff-02c2e232d5e66303434e92c2e77cd64605c3929643b6f553457884d3a0f9d719L35-R44))
* Add the `Button` component as a child of the `EmptyState` component in the `ServiceConnectionsPage` component, with the appropriate props to render a secondary button with a plus icon and a text label, which encourages the user to create a new message broker from the empty state ([link](https://github.com/amplication/amplication/pull/6874/files?diff=unified&w=0#diff-02c2e232d5e66303434e92c2e77cd64605c3929643b6f553457884d3a0f9d719L77-R100))
* Add the `onClick` handler of the button, which uses the `history` variable to push the route for creating a new message broker from the empty state ([link](https://github.com/amplication/amplication/pull/6874/files?diff=unified&w=0#diff-02c2e232d5e66303434e92c2e77cd64605c3929643b6f553457884d3a0f9d719L77-R100))

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by Meyazhagan** [comment]
Screenshot
image

File: packages/amplication-client/src/ServiceConnections/ServiceConnectionsPage.tsx [hunk 1]
```diff
@@ -74,7 +82,22 @@ const TopicsPage: React.FC<Props> = ({ match, innerRoutes }: Props) => {
...
+                `/${currentWorkspace?.id}/${currentProject?.id}/create-broker`
+              )
+            }
+            type="button"
+            buttonStyle={EnumButtonStyle.Secondary}
+            icon="plus"
+            iconPosition={EnumIconPosition.Left}
+            iconSize="xsmall"
+          >
+            <span style={{ color: "var(--black100)" }}>Add Message Broker</span>
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/ServiceConnections/ServiceConnectionsPage.tsx:98]: please use the CSS file for the style

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
looks good! with one minor comment

File: packages/amplication-client/src/ServiceConnections/ServiceConnectionsPage.tsx (hunk 1 above)
Comment (Meyazhagan) [reply packages/amplication-client/src/ServiceConnections/ServiceConnectionsPage.tsx]: @yuval-hazaz Done the changes

**Pull request merged**